### PR TITLE
Feature page 6

### PR DESCRIPTION
**Pull request by bauHau**

### CODE REVIEW

File: page_6.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
 	</head>
 	<body>
 		<h1 class="text-center">Page 6</h1>
+		<a href="">Home</a>
```
Comment (fullStackDevs) [inline page_6.html]: This should be have a "btn btn-primary" class.

File: page_6.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 		<![endif]-->
 	</head>
 	<body>
-		<h1 class="text-center">Hello World</h1>
+		<h1 class="text-center">Page 6</h1>
```
Comment (fullStackDevs) [inline page_6.html]: You should use an h2 tag.

**Review by fullStackDevs — CHANGES_REQUESTED** [review]

File: page_6.html [hunk 3]
```diff
@@ -23,6 +23,12 @@ <h1 class="text-center">Page 6</h1>
 
 		<section>
 			<h3>Section one</h3>
+			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```
Comment (bauHau) [inline page_6.html]: This paragraph should be inside a container section.

File: page_6.html [hunk 4]
```diff
@@ -19,6 +19,7 @@
 	<body>
 		<h1 class="text-center">Page 6</h1>
 		<a href="">Home</a>
+		<a href="">About</a>
```
Comment (bauHau) [inline page_6.html]: This link should have the class "btn btn-primary"

**Review by bauHau — COMMENTED** [review]
Please make the necessary modifications before merging into master.

**Comment by bauHau** [comment]
The code looks ok now.

**Comment by fullStackDevs** [comment]
Now it looks goot.

**Review by fullStackDevs — APPROVED** [review]
This looks ok---

**Review by bauHau — COMMENTED** [review]
Good to go.

**Comment by fullStackDevs** [comment]
Closing the pull request. The merge has been done from the command line.

**Pull request closed without merging**